source/frontend/src/hooks/repositories/useManagementActivityRepository.ts

```diff
@@ -48,6 +49,18 @@ export const useManagementActivityRepository = () => {
     onError: useAxiosErrorHandler('Failed to create a management file activity.'),
   });
 
+  const getPropertyManagementActivity = useApiRequestWrapper<
+    (propertyActivityId: number) => Promise<AxiosResponse<ApiGen_Concepts_PropertyActivity, any>>
+  >({
+    requestFunction: useCallback(
+      async (propertyActivityId: number) => await getPropertyActivityApi(propertyActivityId),
```

Comment:
is this getting a management activity or just getting an activity regardless of context?

Reply:
any activity, regardless if its in a managementfile context